src/main/resources/META-INF/plugin.xml

```diff
@@ -3,7 +3,7 @@
     <name>Git Machete</name>
     <vendor email="gitmachete@virtuslab.com" url="https://virtuslab.com">VirtusLab</vendor>
 
-    <depends>com.intellij.modules.externalSystem</depends>
+    <depends optional="true" config-file="com.virtuslab.git-machete-withExternalSystem.xml">com.intellij.modules.externalSystem</depends>
```

Comment:
Let's see if that satisfies the Marketplace... I don't know if there's any way to know that upfront ☹️